angular/projects/researchdatabox/form/src/app/component/radio-input.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Input } from '@angular/core';
+import { FormFieldBaseComponent, FormFieldCompMapEntry, FormFieldModel } from "@researchdatabox/portal-ng-common";
+import { RadioInputComponentConfig } from '@researchdatabox/sails-ng-common';
+import { get as _get, isEmpty as _isEmpty, isUndefined as _isUndefined } from 'lodash-es';
+
+export class RadioInputModel extends FormFieldModel<string> {
+}
+
+export interface RadioOption {
```

Comment:
The interface for CheckboxOption and DropdownOption are essentially the same as RadioOption. Could they be collapsed to one interface?

Also, why is value `any`? [Shouldn't it be `string`? ](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/radio#value)

Also, this interface needs to be in `sails-ng-common`, so it can be used on the server-side and the client-side.